### ISSUE TITLE
case insensitivity for unicode characters

### ISSUE THREAD

**Issue by tuananh** [RedisLabsModules/RediSearch#291]
How to produce

```
FT.CREATE testIndex SCHEMA name TEXT
FT.ADD testIndex doc1 1.0 FIELDS name "Đà Nẵng"
FT.SEARCH testIndex "đà" # no result
FT.SEARCH testIndex "Đà" # ok
```

I was expecting `FT.SEARCH testIndex "đà"` to works too

**Comment by dvirsky** [comment]
This is a known issue but we won't attend to it in the immediate future, I suggest you normalize the input on the client side if this is urgent. 
 
I'm keeping this open as we do intend to fix this, it just won't be in the next few weeks.

**Comment by tuananh** [comment]
Yes, i'm normalizing data before inputting into redis right now; just wondering if this is expected behavior.

Can you help pointing me to the section where i need to make the changes to fix this?

**Comment by dvirsky** [comment]
It's not a trivial fix, but if you want to explore it, https://github.com/RedisLabsModules/RediSearch/blob/master/src/tokenize.c#L39

**Comment by tw-bert** [comment]
I guess, to address this properly, an OSS collation module should be incorporated into RediSearch.

**Comment by dvirsky** [comment]
@tw-bert it's actually there, an excellent but little known tiny unicode library called libnu. It's just not used in the tokenizer. It is used for other things.

**Comment by tw-bert** [comment]
@dvirsky : Great info, and libnu indeed has collation functions. We have a need for collation as well, but no immediate need. Good to hear it's on the roadmap.
